src/main/resources/jenkins/plugins/slack/SlackNotifier/help-artifactIncludes.html

```diff
@@ -0,0 +1 @@
+ setting that specifies the generated raw XML report files, such as ‘myproject/target/test-reports/*.xml’. Basedir of the fileset is <a href="http://ant.apache.org/manual/Types/fileset.html">Fileset ‘includes’ </a><a href="ws/">the workspace root</a>.
```

Comment:
this doesn't really make sense to me, can you rephrase it possibly?

Reply:
We still need some help for the upload field.